lib/backends/webgl/ops/conv-pack.ts

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import {Attribute} from '../../../attribute';
+import {Logger} from '../../../instrument';
+import {Conv} from '../../../ops/conv';
+import {Tensor} from '../../../tensor';
+import {PoolConvUtil} from '../../../util';
+import {WebGLInferenceHandler} from '../inference-handler';
+import {Artifact, ProgramInfo} from '../types';
+import {WebGLConv} from './conv';
+import {WebGLIm2ColPacked} from './im2col-pack';
+import {WebGLMatMulPacked} from './matmul-pack';
+import {WebGLReshapePacked} from './reshape-packed';
+
+export class WebGLConvPacked extends Conv {
+  unpackedImpl: WebGLConv;
+  constructor() {
+    super();
+    this.unpackedImpl = new WebGLConv();
```

Comment:
nit: it is a bit weird to have a packed conv including a unpacked conv impl as member variable. Maybe we can add a comment here and also rename it to fallbackImpl?

Reply:
part of the refactoring aforenoted.